src/main/java/seedu/address/logic/parser/FindStarsLessThanCommandParser.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.FindStarsLessThanCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.StarsLessThanPredicate;
+
+/**
+ * Parser for the FindStarsLessThanCommand.
+ */
+public class FindStarsLessThanCommandParser implements Parser<FindStarsLessThanCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindStarsLessThanCommand
+     * and returns an FindStarsLessThan object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public FindStarsLessThanCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        try {
+            String[] splitArgs = args.split(" ");
+            int intValue = Integer.parseInt(splitArgs[1]);
+            if (intValue < 1 || splitArgs.length > 2) {
+                throw new ParseException(FindStarsLessThanCommand.MESSAGE_VALID_UPPER_BOUND);
+            }
+            StarsLessThanPredicate predicate = new StarsLessThanPredicate(intValue);
+            return new FindStarsLessThanCommand(predicate);
```

Comment:
the `NumberFormatException | ArrayIndexOutOfBoundsException` will never(?) be thrown here, so I think it's better to move these lines to after the try block.

Reply:
Would you suggest I initialize intValue before the try block so I can move these lines out?